web/collections.textile

```diff
@@ -185,14 +185,14 @@ scala> numbers.map((i: Int) => i * 2)
 res0: List[Int] = List(2, 4, 6, 8)
 </pre>
 
-or pass in a partially evaluated function
+or pass in a function
```

Comment:
What do you think about adding a parenthesis here?

"or pass in a function (the Scala compiler automatically converts our method to a function)"